src/super_gradients/recipes/dataset_params/cifar100_dataset_params.yaml

```diff
@@ -0,0 +1,54 @@
+batch_size: 256 # batch size for trainset
+val_batch_size: 512 # batch size for valset in DatasetInterface
+
+# TODO: REMOVE ABOVE, HERE FOR COMPATIBILITY UNTIL WE REMOVE DATASET_INTERFACE
+
+train_dataset_params:
+  root: /data/cifar100
+  train: True
+  transforms:
+    - RandomCrop:
+        size: 24
+        padding: 4
+    - RandomHorizontalFlip
+    - ToTensor
+    - Normalize:
+        mean:
+          - 0.4914
```

Comment:
These are the default transforms for cifar10 not cifar100 (take a look at the corresponding dataset interface class and their default parameters defined in LIBRARY_DATASETS).